rpcserver.go

```diff
@@ -987,7 +994,7 @@ func allowCORS(handler http.Handler, origins []string) http.Handler {
 func (r *rpcServer) sendCoinsOnChain(paymentMap map[string]int64,
 	feeRate chainfee.SatPerKWeight, label string) (*chainhash.Hash, error) {
 
-	outputs, err := addrPairsToOutputs(paymentMap)
+	outputs, err := addrPairsToOutputs(paymentMap, r.rpcNetParams.Params)
```

Comment:
Can just use `r.cfg.ActiveNetParams` here and below.

Reply:
yup